src/Internal/DependencyManagement/CoreServiceProvider.php

```diff
@@ -311,6 +318,10 @@ function ( ...$arguments ) {
 		$this->conditionally_share_with_tags( CouponChannelVisibilityMetaBox::class, Admin::class, CouponMetaHandler::class, CouponHelper::class, MerchantCenterService::class );
 		$this->conditionally_share_with_tags( MetaBoxInitializer::class, Admin::class, MetaBoxInterface::class );
 
+		// Share bulk edit views
+		$this->conditionally_share_with_tags( CouponBulkEdit::class, CouponMetaHandler::class, MerchantCenterService::class );
+		$this->conditionally_share_with_tags( BulkEditInitializer::class );
```

Comment:
Neither of these implement the `Conditional` class, so they can use `share_with_tags`
```suggestion
		$this->share_with_tags( CouponBulkEdit::class, CouponMetaHandler::class, MerchantCenterService::class );
		$this->share_with_tags( BulkEditInitializer::class );
```